sdk/cdn/arm-cdn/package.json

```diff
@@ -2,11 +2,11 @@
   "name": "@azure/arm-cdn",
   "author": "Microsoft Corporation",
   "description": "CdnManagementClient Library with typescript type definitions for node.js and browser.",
-  "version": "4.2.0",
+  "version": "4.3.0",
```

Comment:
We should increment the major version and not the minor here. Since the ms-rest-azure dependencies are changing to v2, users would require to move to v3 of ms-rest-nodeauth to do the auth

cc @qiaozha

Reply:
fixed